.github/workflows/backend-test-and-build.yml

```diff
@@ -7,11 +7,11 @@ on:
       - completed
```

Comment:
Might need to do another pull from 4716

Reply:
Cheers, yep will repoint the PR once 4716 merged to main,